### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by maksym-nyrka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,27 +64,30 @@ docker run \
     --telemetry-url "wss://test.telemetry.sydney.ggxchain.io/submit 0"
 ```
 
-#### Testnet
+
+### Brooklyn
```
Comment (uchu) [inline README.md:68]: (Development chain)

**Review by uchu — APPROVED** [review]
LGTM

**Review by obsessed-cake — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -164,3 +123,15 @@ nix run .#single-fast # to run an one node network
 nix run .#multi-fast # to run 3-node network
 nix run .#prune-running # to stop nodes
 ```
+
+
+
+##### If you want to compile WASM contracts, you'll need additional dependencies:
+```bash
+# Install wasm32-unknown-unknown target
+rustup target add wasm32-unknown-unknown
```
Comment (akorchyn) [inline README.md:132]: I believe this we still need to compile runtime.

**Review by akorchyn — APPROVED** [review]
Small nitpick, other than that well done

**Pull request merged**